<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MongoJobWidget</class>
 <widget class="QWidget" name="MongoJobWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>981</width>
    <height>781</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MongoJobWidget</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>2</number>
   </property>
   <property name="topMargin">
    <number>2</number>
   </property>
   <property name="rightMargin">
    <number>2</number>
   </property>
   <property name="bottomMargin">
    <number>2</number>
   </property>
   <item>
    <widget class="QProgressBar" name="loadingProgressBar">
     <property name="value">
      <number>0</number>
     </property>
     <property name="textVisible">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QCheckBox" name="serviceJobCheckBox">
       <property name="text">
        <string>Service jobs</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="userJobCheckBox">
       <property name="text">
        <string>User jobs</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="successCheckBox">
       <property name="text">
        <string>Success</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="cancelCheckBox">
       <property name="text">
        <string>Canceled</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="failCheckBox">
       <property name="text">
        <string>Failed</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="filterLineEdit"/>
     </item>
     <item>
      <widget class="QPushButton" name="applyFilterPushButton">
       <property name="toolTip">
        <string>Apply filter</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="plugin.qrc">
         <normaloff>:/plugin/images/Resources/apply.png</normaloff>:/plugin/images/Resources/apply.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <widget class="QTreeView" name="jobsTreeView">
      <property name="alternatingRowColors">
       <bool>true</bool>
      </property>
      <property name="sortingEnabled">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QToolBox" name="jobDetailsToolBox">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="page">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>977</width>
         <height>323</height>
        </rect>
       </property>
       <attribute name="label">
        <string>General information</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0">
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Summary</string>
          </property>
         </widget>
        </item>
        <item row="0" column="2" colspan="5">
         <widget class="QLineEdit" name="summaryLineEdit">
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="2" column="2">
         <widget class="QLineEdit" name="statusLineEdit">
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="2" column="4" colspan="3">
         <widget class="QLineEdit" name="phaseLineEdit">
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="2" column="3">
         <widget class="QLabel" name="label_8">
          <property name="text">
           <string>Phase</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>ID</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Status</string>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QLabel" name="label_9">
          <property name="text">
           <string>Creation time</string>
          </property>
         </widget>
        </item>
        <item row="4" column="6">
         <widget class="QLineEdit" name="endLineEdit">
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="4" column="2">
         <widget class="QLineEdit" name="creationLineEdit">
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="4" column="4">
         <widget class="QLineEdit" name="startLineEdit">
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="1" column="2" colspan="5">
         <widget class="QLineEdit" name="idLineEdit">
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="4" column="3">
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>Start time</string>
          </property>
         </widget>
        </item>
        <item row="4" column="5">
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>End time</string>
          </property>
         </widget>
        </item>
        <item row="5" column="4">
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="page_2">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>98</width>
         <height>165</height>
        </rect>
       </property>
       <attribute name="label">
        <string>Error information</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QSplitter" name="splitter_2">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <widget class="QTextBrowser" name="errorDetailsTextBrowser"/>
          <widget class="QTextBrowser" name="errorStackTextBrowser"/>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="plugin.qrc"/>
 </resources>
 <connections/>
</ui>
